Checkout/Source/Tokenisation/CheckoutAPIService.swift

```diff
@@ -126,12 +149,21 @@ final public class CheckoutAPIService: CheckoutAPIProtocol {
       requestParameters,
       responseType: TokenResponse.self,
       responseErrorType: TokenisationError.ServerError.self
-    ) { [tokenDetailsFactory, logManager, logTokenResponse] tokenResponseResult, httpURLResponse in
+    ) { [riskSDK, tokenDetailsFactory, logManager, logTokenResponse] tokenResponseResult, httpURLResponse in
```

Comment:
```suggestion
    ) { [weak riskSDK, tokenDetailsFactory, logManager, logTokenResponse] tokenResponseResult, httpURLResponse in
```

I believe we need to get the RiskSDK instance as weakified here to avoid the retain cycle where we call publishData inside the configure's completion closure a couple of lines below.